### PR TITLE
Add fleet management diagram to docs

### PR DESCRIPTION
**Pull request by npentrel**
# Description

Please include a summary of the change and what is does. Please also include relevant motivation and context.

### CODE REVIEW

File: docs/fleet/_index.md [hunk 1]
```diff
@@ -24,14 +24,38 @@ You can manage and control your fleet of {{< glossary_tooltip term_id="machine"
...
+<!-- markdownlint-disable MD001 -->
 
-<!-- TODO: Add topology based on Jon's draft -->
+{{< cards >}}
+{{% manualcard link="/fleet/organizations/" %}}
 
-![An image of two locations, New York, and Chicago, in one organization, Good Robots](/fleet/locations.png)
+#### Organizations
+
+The highest level grouping, generally used for different companies
```
Comment (andf-viam) [inline docs/fleet/_index.md:34]:
```suggestion
The highest level grouping, generally used for different companies.
```

File: docs/fleet/_index.md [hunk 2]
```diff
@@ -24,14 +24,38 @@ You can manage and control your fleet of {{< glossary_tooltip term_id="machine"
...
+#### Locations
+
+Virtual groupings of devices up with up to three levels of nesting that can represent a grouping of machines that are co-located in a building, like a factory, or a grouping of machines that are thousands of miles apart and are grouped together by function or as an organizational unit.
+
+{{% /manualcard %}}
+{{% manualcard link="/fleet/machines/" %}}
+
+#### Machines
+
+A grouping of {{< glossary_tooltip term_id="component" text="components" >}} and {{< glossary_tooltip term_id="service" text="services" >}} across one {{< glossary_tooltip term_id="part" text="part" >}}, or multiple parts working closely together to complete tasks.
```
Comment (andf-viam) [inline docs/fleet/_index.md:48]:
```suggestion
A grouping of {{< glossary_tooltip term_id="component" text="components" >}} and {{< glossary_tooltip term_id="service" text="services" >}} across one {{< glossary_tooltip term_id="part" text="part" >}} or more parts working closely together to complete tasks.
```

File: docs/fleet/_index.md [hunk 3]
```diff
@@ -24,14 +24,38 @@ You can manage and control your fleet of {{< glossary_tooltip term_id="machine"
...
+
+The highest level grouping, generally used for different companies
+
+{{% /manualcard %}}
+{{% manualcard link="/fleet/locations/" %}}
+
+#### Locations
+
+Virtual groupings of devices up with up to three levels of nesting that can represent a grouping of machines that are co-located in a building, like a factory, or a grouping of machines that are thousands of miles apart and are grouped together by function or as an organizational unit.
+
```
Comment (andf-viam) [inline docs/fleet/_index.md:42]: Opt: I like the machines clarification on line 49, here's a similar one for loc:

```suggestion

An organization can have multiple locations.

```

But it's also definitely clear from the diagram. This for searchers, and textonly-ers, I suppose. Definitely optional!

**Review by andf-viam — APPROVED** [review]
LGTM! Some optional thoughts if interested!

File: docs/fleet/_index.md [hunk 4]
```diff
@@ -24,14 +24,39 @@ You can manage and control your fleet of {{< glossary_tooltip term_id="machine"
 
 To organize your fleet you use
```
Comment (JessamyT) [inline docs/fleet/_index.md:25]:
```suggestion
To organize your fleet you use:
```
[nit] Sorry for drive-by....I've been wanting to add this colon but see that this PR is in the works so putting it in here instead of making separate PR!

File: docs/fleet/_index.md [hunk 5]
```diff
@@ -24,14 +24,39 @@ You can manage and control your fleet of {{< glossary_tooltip term_id="machine"
...
+#### Organizations
+
+The highest level grouping, generally used for different companies.
+
+{{% /manualcard %}}
+{{% manualcard link="/fleet/locations/" %}}
+
+#### Locations
+
+Virtual groupings of devices up with up to three levels of nesting that can represent a grouping of machines that are co-located in a building, like a factory, or a grouping of machines that are thousands of miles apart and are grouped together by function or as an organizational unit.
```
Comment (JessamyT) [inline docs/fleet/_index.md:41]:
```suggestion
A virtual grouping of devices up with up to three levels of nesting that can represent a grouping of machines that are co-located in a building, like a factory, or a grouping of machines that are thousands of miles apart and are grouped together by function or as an organizational unit.
```
[nit] consistency
